pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1304,6 +1304,15 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 		return nil, err
 	}
 
+	// set drivers cache mode
+	for i := range domain.Spec.Devices.Disks {
```

Comment:
Can you create a dedicated function for this? Looks like a good place wold still be in the converter package. We should probably also add a few unit tests.

Reply:
Each loop has different return elements. With the amount of non DRY between SyncVMI and PrepareMigrationTarget does it matter?

Reply:
> Each loop has different return elements.

Not sure I understand. I thought about putting the whole block in one function. It makes is easier to discover where we set the cache already, or not?

Reply:
How is this?

Reply:
?

Reply:
@rmohr